tuxemon/event/actions/add_monster.py

```diff
@@ -76,3 +76,4 @@ def start(self) -> None:
         current_monster.current_hp = current_monster.hp
 
         trainer.add_monster(current_monster)
+        trainer.tuxepedia[monster_slug] = 2
```

Comment:
Can we use a string enum?

```
class SeenStatus(str, Enum):
    seen = "seen"
    caught = "caught"
```

Reply:
I am also considering counting the number of times you have seen / caught instead. There is a slight complication in that you might want to not count an NPC's monster multiple times, but we don't need to support this straight away.

What do you think?

Reply:
I'm open to your proposal. I think enum is a good concept.

Reply:
Let's go with counting then.